src/site/content/en/blog/components-howto-tabs/index.md

```diff
@@ -207,6 +213,13 @@ Give each panel a `aria-labelledby` attribute that refers to the tab that contro
 
 The element checks if any of the tabs have been marked as selected. If not, the first tab is now selected.
 
+```js
+      const selectedTab =
+        tabs.find((tab) => tab.selected) || tabs[0];
+```
+
+Next, switch to the selected tab. `_selectTab()` takes care of marking all other tabs as deselected and hiding all other panels.
```

Comment:
See the [source](https://github.com/GoogleChromeLabs/howto-components/blob/079d0fa34ff9038b26ea8883b1db5dd6b677d7ba/elements/howto-tabs/howto-tabs.js#L161-L165).